sympy/geometry/line.py

```diff
@@ -1083,24 +1093,68 @@ class Line(LinearEntity):
     >>> s = Segment((0, 0), (0, 1))
     >>> Line(s).equation()
     x
+
+    >>> # a circle object is returned
```

Comment:
@smichr Done. Please take a look.

Reply:
I still see the circle-reference in this Line class docstring. And please make it a sentence instead of an inline comment. The examples can be commented on:
```python
A line can be instantiated with an expression:

>>> from sympy.abc import x, y
>>> Line(3*x + y + 18)
Line2D(Point2D(0, -18), Point2D(1, -21))
```

note also that you don't need a & b (at least from what I see here)